main.go

```diff
@@ -59,38 +69,114 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 		// Unmarshal JSON
 		b, err := ioutil.ReadAll(r.Body)
 		if err != nil {
-			http.Error(w, "Invalid request method", http.StatusBadRequest)
+			raven.CaptureError(err, nil)
+			http.Error(w, "Invalid request", http.StatusBadRequest)
 			return
 		}
 		defer r.Body.Close()
 		req := indexRequest{}
 		json.Unmarshal(b, &req)
 
-		// Convert []Trytes to []Transaction
-		txs := make([]giota.Transaction, len(req.Trytes))
-		for i, t := range req.Trytes {
-			tx, _ := giota.NewTransaction(t)
-			txs[i] = *tx
-		}
+		go attachAndBroadcastToTangle(&req)
+
+		w.WriteHeader(http.StatusNoContent)
+	} else {
+		http.Error(w, "Invalid request method", http.StatusMethodNotAllowed)
+	}
+}
+
+func attachAndBroadcastToTangle(indexReq *indexRequest) {
+	provider := os.Getenv("PROVIDER")
+	// minDepth, _ := strconv.ParseInt(os.Getenv("MIN_DEPTH"), 10, 64)
```

Comment:
`attachToTangle` doesn't take this in as a param. I don't know if they pick one for us?